<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormNovaAquisicao</class>
 <widget class="QDialog" name="FormNovaAquisicao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame1, #frame2 {
	background-color: #e6e6e6;
}

QSpinBox, QDateEdit, QTimeEdit {
	border: 1px solid #bfbebd;
	border-radius: 10px;
	padding: 2px;
}

QSpinBox::up-button  {
	subcontrol-origin: margin;
	subcontrol-position: top right;
	right: 5px;
}
QSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: bottom right;
	right: 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QLabel" name="titulo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;div style=&quot;text-align: center;&quot;&gt;&lt;h1&gt;Registrar aquisição&lt;/h1&gt;&lt;/div&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Código do fornecedor:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pesquisarFornButton">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Pesquisar fornecedor</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 3px;
border: none;
border-radius: 8px;
background-color: #b1b1af;
}

:hover {
background-color: #a8a8a6;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Recursos.qrc">
         <normaloff>:/imgs/imagens/buscarProduto.png</normaloff>:/imgs/imagens/buscarProduto.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSpinBox" name="codForn">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame1">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Data da aquisição:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dataAq">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dataCheckBox">
        <property name="text">
         <string>Usar data atual</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hora da aquisição:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="horaAq">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="horaCheckBox">
       <property name="text">
        <string>Usar hora atual</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Código do produto:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pesquisarProdButton">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pesquisar produto</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding: 3px;
border: none;
border-radius: 8px;
background-color: #b1b1af;
}

:hover {
background-color: #a8a8a6;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Recursos.qrc">
          <normaloff>:/imgs/imagens/buscarProduto.png</normaloff>:/imgs/imagens/buscarProduto.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSpinBox" name="codProd">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Quantidade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="quantidade">
       <property name="maximum">
        <number>999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="registrarButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	margin-top: 5px;
	margin-bottom: 5px;
	padding: 5px;
	border-radius: 10px;
	background-color: #26a844;
	color: #fff;
}

QPushButton:hover {
	background-color: #22973d;
}</string>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
